src/olympia/reviews/models.py

```diff
@@ -256,12 +255,6 @@ def update_denormalized_fields(self):
         pair = self.addon_id, self.user_id
         tasks.update_denorm(pair)
 
-    @staticmethod
-    def transformer(reviews):
```

Comment:
This wasn't used, since nothing is calling `qs.transform(Review.transformer)` anywhere.